src/App.js

```diff
@@ -1,17 +1,21 @@
-import Calculator from './components/Calculator';
-import Header from './components/Header';
+import React from 'react';
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+import Calculator from './components/calculator';
+import Home from './components/Home';
+import Quotes from './components/Quotes';
+import NotFound from './components/notFound';
 
-function App() {
-  return (
-    <div className="App">
-      <header>
-        <Header />
-      </header>
-      <div>
-        <Calculator />
-      </div>
-    </div>
-  );
-}
+const App = () => (
+  <>
+    <Router>
+      <Routes>
+        <Route path="/" index element={<Home />} />
+        <Route path="/calculator" element={<Calculator />} />
+        <Route path="/quotes" element={<Quotes />} />
```

Comment:
Well done ✔ The website contains Home, Calculator, and Quote 🔥 All links are working as expected 🙌